src/devices/Tlc1543/README.md

```diff
@@ -0,0 +1,15 @@
+﻿# Tlc1543
+
+## Summary
+High-speed 10-bit switched-capacitor successive-approximation A/D Converter with 14 channels (11 inputs and 3 self-test channels)
+
+## Device Family
+**[TLC1542/3]**: [Datasheet](https://www.ti.com/lit/ds/symlink/tlc1543.pdf)
+
+## Binding Notes
+(For now) Implemented Fast Mode 1 which doesn't need EOC pin connected. Respective timing diagram can be seen on figure 9 in datasheet.
+
+It is possible to change ADC charge channel.
```

Comment:
Maybe add as well how to create the ADC element and how to read? I understand it's just super easy and straight forward but those little things helps a lot.